src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/version/GerritVersionChecker.java

```diff
@@ -66,7 +66,17 @@
         /**
          * Gerrit CommentAdded always contains approval information, added in Gerrit 2.13.
          */
-        commentAlwaysApproval("CommentAdded always contains approval", "2.13");
+        commentAlwaysApproval("CommentAdded always contains approval", "2.13"),
+
+        /**
+         * WIPStateChanged events, added in Gerrit 2.15.
+         */
+        wipStateChanged("WIPStateChanged events", "2.15"),
+
+        /**
+         * PrivateStateChanged events, added in Gerrit 2.15.
+         */
+        privateStateChanged("privateStateChanged events", "2.15");
```

Comment:
You marked @sselberg 's comment as resolved but I don't see you using it anywhere?
In this case I don't think we will need version checks but then that should be explained in the docs I think.

Reply:
I added some dummy checks against the comments which I removed them by skipping those commits. I misunderstood the comments which should be a remind instead of a "must". Going to remove those two (wipStateChanged & privateStateChanged) checks in this PR.